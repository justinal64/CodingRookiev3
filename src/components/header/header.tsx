import React from "react";

interface Props {
  name: string;
}

export default function Welcome(props: Props) {
  return <h1>Hello, {props.name}</h1>;
}
