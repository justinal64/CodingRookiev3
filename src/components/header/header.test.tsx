import React from "react";
import ReactDOM from "react-dom";
import { Header } from "./header";
import { shallow } from "enzyme";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Header name="test" />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("should render banner text correctly with given strings", () => {
  const component = shallow(<Header name="testers" />);
  expect(component).toMatchSnapshot();
});

// write test that test the values being passed in!!!!!!
