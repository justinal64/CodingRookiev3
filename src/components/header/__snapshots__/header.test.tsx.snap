// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render banner text correctly with given strings 1`] = `
<h1>
  Hello, 
  tester
</h1>
`;
